### PR TITLE
Feat/rl-3/frontend-init

### PR DESCRIPTION
**Pull request by ahureikinandrei**
1. React
1 React
2. Eslint, prettier
2 Elint,prettier
3. App component
3 App
4. Header (header)
4 Header
5. Footer (footer)
5 Footer
6. Main (main)
6 Main

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (aadereiko) [inline client/public/index.html]: change the title of the app

File: client/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (aadereiko) [inline client/public/manifest.json]: change the manifest file according to your app

**Review by aadereiko — APPROVED** [review]
Impressive work! I have nothing important to pick on. 
some small details in general settings

File: client/public/index.html (hunk 1 above)
Comment (ahureikinandrei) [reply client/public/index.html]: fix

File: client/public/manifest.json (hunk 2 above)
Comment (ahureikinandrei) [reply client/public/manifest.json]: fix

**Pull request merged**